frontend/src/app/shared/classes/app.config.ts

```diff
@@ -1,11 +1,13 @@
 import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
 import { CustomtextService } from '../services/customtext/customtext.service';
+// eslint-disable-next-line import/no-relative-packages
 import customTextsDefault from '../../../../../definitions/custom-texts.json';
 import { KeyValuePairs } from '../../app.interfaces';
 import {
   AppSettings,
   BroadCastingServiceStatus,
-  localStorageTestConfigKey, standardBackgroundBody, standardBackgroundBox,
+  standardBackgroundBody,
```

Comment:
Konstanten werden üblicherweise in allcaps geschrieben. Ich sehe, dass du damit nicht angefangen hast, wäre aber trotzdem angebracht das mal ordentlich zu machen.